examples/siro_sandbox/server/marco_polo_client.py

```diff
@@ -0,0 +1,20 @@
+import asyncio
```

Comment:
marco_polo_client and marco_polo_server are a minimal websocket-based client/server example. They are here for reference; they aren't used by the HITL tool.